jwql/instrument_monitors/miri_monitors/cosmic_ray_monitor.py

```diff
@@ -0,0 +1,467 @@
+                                                                cosmic_ray_db_entry = {'entry_date': datetime.datetime.now(),
+                                                                    'aperture': aperture,
+                                                                        'source_file': file_name,
+                                                                            'obs_start_time': start_time,
+                                                                                'obs_end_time': end_time,
+                                                                                    'jump_count': cosmic_ray_num,
+                                                                                        'magnitude': cosmic_ray_mags
+                                                                                        }
+                                                                                            stats_table.__table__.insert().execute(cosmic_ray_db_entry)
+                                                                                            
+                                                                                                logging.info("Successfully inserted into database. \n")
+                                                                                                    except:
+                                                                                                        logging.info("Could not insert entry into database. \n")
+
+def get_cr_mags(jump_locs, jump_locs_pre, rateints, jump_data, jump_head, nints):
+    """
+        Computes a list of magnitudes using the coordinate of the detected jump compared to
+        the magnitude of the same pixel in the group prior to the jump.
+        
+        Parameters:
+        ----------
+        
+        jump_locs: list
+        List of coordinates to a pixel marked with a jump.
+        
+        jump_locs_pre: list
+        List of matching coordinates one group before jump_locs.
+        
+        head: FITS header
+        Header containing file information.
+        
+        rateints: ndarray
+        Array in DN/s.
+        
+        Returns:
+        -------
+        
+        mags: list
+        A list of cosmic ray magnitudes corresponding to each jump.
+        
+        """
+    
+    mags = []
+    for coord, coord_gb in zip(jump_locs, jump_locs_pre):
+        mags.append(magnitude(coord, coord_gb, rateints, jump_data, jump_head, nints))
+    
+    return mags
+
+def get_cr_rate(jump_locs, t):
+    """
+        Computes the rate of cosmic ray impacts in a given time.
+        
+        Parameters:
+        ----------
+        jump_locs: list
+        List of coordinates to a pixel marked with a jump.
+        
+        t: int or float
+        Time over which to compute the rate.
+        Nominally the effective exposure time.
+        
+        """
+    
+    return len(jump_locs)/t
```

Comment:
I think PEP8 suggests to avoid use of single-character variable names.  I would suggest to use `time` here.

Reply:
changed to `time`